Tools/gdb/libpython.py

```diff
@@ -1502,8 +1502,10 @@ def is_python_frame(self):
         return False
 
     def is_evalframeex(self):
-        '''Is this a PyEval_EvalFrameEx frame?'''
-        if self._gdbframe.name() == 'PyEval_EvalFrameEx':
+        '''Is this a PyEval_EvalFrameEx or _PyEval_EvalFrameDefault (PEP 0523)
+        frame?'''
+        if self._gdbframe.name() in ('PyEval_EvalFrameEx',
```

Comment:
Should `PyEval_FrameEx` even stay as an option if this simply won't work if someone sets a eval frame function? I mean if the gdb support is so reliant on the specifics of `PyEval_EvalFrameDefault` then maybe it isn't worth trying to support `PyEval_EvalFrameEx`?